extras/craft/README.md

```diff
@@ -9,17 +9,17 @@ Include the proper gulp-revisioned version of an asset in your Twig templates wi
 <link rel="stylesheet" href="{{ 'assets/stylesheets/app.css' | gulp_rev }}">
 ```
 
-
-The output of the Twig filter defaults to the given string/path unless `rev-manifest.json` exists. When `rev-manifest.json` exists, then the string is replaced with the revisioned asset name stored in that file.
+The output of the Twig filter defaults to the given string/path unless `rev-manifest.json` exists. When `rev-manifest.json` exists, then the string is replaced with the revisioned asset path.
 
 
 ###Important Notes:
-* The plugin assumes that your `rev-manifest.json` file is in the public folder.
-* Make sure `rev-manifest.json` does not exist when running `npm run gulp` (the development task). This will cause an error since the revisioned file will not exist.
-* Asset paths output by the Twig filter are forced to start with a "/" if it doesn't already exist.
+* By default, the plugin looks for your `rev-manifest.json` file in the public folder.
+* To customize the path to your `rev-manifest.json`, go to `/admin/settings/plugins/gulprev` in the Craft Admin and change the setting for the file path.
+* Make sure `rev-manifest.json` does not exist when running development environment task (when files are not versioned). This will cause an error since the versioned file will not exist.
```

Comment:
I believe you can delete this line, as the file should get deleted when that task gets run